src/Backend/Routes/AuthProviders/GoogleAuth.js

```diff
@@ -0,0 +1,21 @@
+import { getAuth, signInWithPopup, GoogleAuthProvider } from "firebase/auth";
```

Comment:
Same as above, also why so many sign in methods? The only thing that changes is the provider right

Reply:
Could just have parent function called `signInWithProvider(provider: 'Github' | 'Google') => ....`